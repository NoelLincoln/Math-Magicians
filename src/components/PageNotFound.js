import React from 'react';

const PageNotFound = () => (
  <div className="pagenotfound">
    <p>Sorry, Page Not Found</p>
  </div>
);

export default PageNotFound;
