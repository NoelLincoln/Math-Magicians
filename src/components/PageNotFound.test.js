import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import PageNotFound from './PageNotFound';

test('PageNotFound component renders correctly', () => {
  render(<PageNotFound />);

  const pageNotFoundText = screen.getByText('Sorry, Page Not Found');
  expect(pageNotFoundText).toBeInTheDocument();
});

test('PageNotFound component matches snapshot', () => {
  const { asFragment } = render(<PageNotFound />);

  expect(asFragment()).toMatchSnapshot();
});
