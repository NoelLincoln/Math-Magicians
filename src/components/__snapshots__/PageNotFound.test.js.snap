// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="pagenotfound"
  >
    <p>
      Sorry, Page Not Found
    </p>
  </div>
</DocumentFragment>
`;
